pkg/docker/fast_push.go

```diff
@@ -55,6 +57,10 @@ func userAgent() string {
 
 func FastPush(ctx context.Context, image string, projectDir string, command Command) error {
 	g, _ := errgroup.WithContext(ctx)
+	p := mpb.New(
+		mpb.WithWidth(60),
```

Comment:
Why did we decide 60 columns? Are the defaults bad?

In general I'd expect the library to do the right thing if I resize my terminal (if, say, I go below 60 columns wide), so I'm a little suspicious of customizing to a hardcoded value.

(This isn't a blocking issue)

Reply:
This is a cargo cult, I'll remove the width requirement.